core/drivers/crypto/caam/include/caam_desc_defines.h

```diff
@@ -661,6 +682,11 @@
 #define BLACK_KEY_NONCE_SIZE	6
 #define BLACK_KEY_ICV_SIZE	6
 
+/* DL Keypair Generation */
+#define PDB_DL_KEY_L_SIZE(len) SHIFT_U32((len) & (0x3FF), 7)
+#define PDB_DL_KEY_N_MASK      0x7F
+#define PDB_DL_KEY_N_SIZE(len) SHIFT_U32((len) & (PDB_DL_KEY_N_MASK), 0)
```

Comment:
can remove parentheses around `0x3FF` and `PDB_DL_N_MASK`.

Reply:
OK